extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/SuccessExecution.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.scheduler;
+
+import java.time.Instant;
+
+/**
+ * This event is fired synchronously and asynchronously when an execution of a scheduled method is successful.
+ */
+public class SuccessExecution {
+
+    // Keep the fields in order to minimize the breaking changes
+    public final String triggerId;
```

Comment:
The same applies here ;-) https://github.com/quarkusio/quarkus/pull/23639/files#r807743630